maxim/models/prompt.py

```
@@ -275,11 +367,19 @@ def default(self, o):
 
 @ dataclass
 class Error():
```

Ruff (0.11.9)

Unnecessary parentheses after class definition

Remove parentheses

(UP039)